Hermes/appscale/hermes/stats/producers/proxy_stats.py

```diff
@@ -229,6 +228,8 @@ class ProxyStats(object):
   backend = attr.ib(metadata={Meta.ENTITY: HAProxyBackendStats})
   servers = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyServerStats})
   listeners = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyListenerStats})
+  servers_count = attr.ib()  # number of servers serving this proxy
+  listeners_count = attr.ib()  # number of listeners serving this proxy
```

Comment:
I see what you mean. It really could be modified to computed value. But I still don't think it worth to do. All stats models are `frozen`, so `attrs` doesn't allow to modify stats objects after it's created. I wouldn't like to maintain "computed" properties inside attrs class which already overrides some magic methods.